<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Monster_Editor</class>
 <widget class="QDialog" name="Monster_Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>857</width>
     <height>639</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>286</width>
              <height>33</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>286</width>
              <height>33</height>
             </size>
            </property>
            <property name="text">
             <string>Die Monster</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_monster">
            <property name="minimumSize">
             <size>
              <width>286</width>
              <height>516</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>286</width>
              <height>600</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="minimumSize">
                 <size>
                  <width>260</width>
                  <height>33</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>256</width>
                  <height>33</height>
                 </size>
                </property>
                <property name="text">
                 <string>Name des Monsters</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_name">
                <property name="minimumSize">
                 <size>
                  <width>260</width>
                  <height>33</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>256</width>
                  <height>33</height>
                 </size>
                </property>
                <property name="maxLength">
                 <number>25</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGraphicsView" name="graphicsView_gesicht">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>200</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>260</width>
                <height>200</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="gesicht">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>256</width>
                <height>31</height>
               </size>
              </property>
              <property name="text">
               <string>Gesicht</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>260</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QPushButton" name="Koerper">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>31</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>256</width>
                <height>31</height>
               </size>
              </property>
              <property name="text">
               <string>Körper</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGraphicsView" name="graphicsView_koerper">
              <property name="minimumSize">
               <size>
                <width>260</width>
                <height>200</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>260</width>
                <height>200</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>13</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>245</width>
            <height>33</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>245</width>
            <height>33</height>
           </size>
          </property>
          <property name="text">
           <string>Werte</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_leben">
              <property name="text">
               <string>Leben</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_mana">
              <property name="text">
               <string>Mana</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_kraft">
              <property name="text">
               <string>Kraft</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_ausdauer">
              <property name="text">
               <string>Ausdauer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_intelligenz">
              <property name="text">
               <string>Intelligenz</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_verteidigung">
              <property name="text">
               <string>Verteidigung</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_glueck">
              <property name="text">
               <string>Glück</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QLineEdit" name="lineEdit_Leben">
              <property name="maxLength">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Mana">
              <property name="maxLength">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_Kraft">
              <property name="maxLength">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_ausdauer">
              <property name="maxLength">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_intelligenz">
              <property name="maxLength">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_verteidigung">
              <property name="maxLength">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_glueck">
              <property name="maxLength">
               <number>10</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>45</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>245</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>245</width>
            <height>10</height>
           </size>
          </property>
          <property name="text">
           <string>Schutz vor in %</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QCheckBox" name="checkBox_eis">
              <property name="text">
               <string>Eis</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_feuer">
              <property name="text">
               <string>Feuer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_wind">
              <property name="text">
               <string>Wind</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_gift">
              <property name="text">
               <string>Gift</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_betaeubt">
              <property name="text">
               <string>Betaeubt</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QSpinBox" name="spinBox_eis">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_feuer">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_wind">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_gift">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_betaeubt">
              <property name="maximum">
               <number>100</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton_monster_neu">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>103</width>
            <height>31</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>103</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Monster neu</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>31</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_monster_entfernen">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>Monster entfern</string>
          </property>
          <property name="autoRepeatDelay">
           <number>300</number>
          </property>
          <property name="autoRepeatInterval">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>550</width>
          <height>31</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>10</y>
     <width>290</width>
     <height>282</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="minimumSize">
         <size>
          <width>286</width>
          <height>33</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>33</height>
         </size>
        </property>
        <property name="text">
         <string>Gegenstände die ein Monster fallen lässt.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listWidget_effekte">
        <property name="minimumSize">
         <size>
          <width>286</width>
          <height>200</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>286</width>
          <height>200</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="pushButton_gegenst_hinzufuegen">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Gegenst. hinzufügen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_gegenstand_entfernen_entfernen">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string>Gegenst. entfernen</string>
        </property>
        <property name="autoRepeatDelay">
         <number>300</number>
        </property>
        <property name="autoRepeatInterval">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_schliessen">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>620</y>
     <width>85</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Schließen</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget8">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>300</y>
     <width>204</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLabel" name="label_geld">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>33</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>33</height>
       </size>
      </property>
      <property name="text">
       <string>Geld</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_geld"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
